File: src/main/java/guru/xgm/language/lni/psi/tree/LniIElement.java
package guru.xgm.language.lni.psi.tree;

import com.intellij.psi.tree.IElementType;
import guru.xgm.language.lni.lang.LniLanguage;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class LniIElement extends IElementType {

    public LniIElement(@NotNull @NonNls String debugName) {
        super(debugName, LniLanguage.INSTANCE);
    }

    @Override
    public String toString() {
        return "LniIElement." + super.toString();
    }
}

File: src/main/kotlin/guru/xgm/language/lni/psi/tree/LniIElement.kt
package guru.xgm.language.lni.psi.tree

import com.intellij.psi.tree.IElementType
import guru.xgm.language.lni.lang.LniLanguage.Companion.instance
import org.jetbrains.annotations.NonNls

class LniIElement(debugName: @NonNls String) : IElementType(debugName, instance) {
    override fun toString(): String = "LniIElement." + super.toString()
}
